build.gradle.kts

```diff
@@ -10,13 +10,13 @@ val prometheusVersion = "0.8.1"
 val micrometerVersion = "1.7.3"
 val spekVersion = "2.0.9"
 val mockkVersion = "1.10.2"
-val slf4jVersion = "1.7.30"
-val logbackVersion = "1.2.3"
+val slf4jVersion = "2.0.0-alpha5"
```

Comment:
`2.0.0-alpha5`, er den stabil da?

Reply:
Kanskje ikke 100 %, men ser den brukes av flere på huset i produksjonskode. Jeg tror det er det eneste som fungerer sammen med den versjonen av Jetty vi trenger for å unngå klager fra Snyk